python/mxnet/gluon/estimator/event_handler.py

```diff
@@ -98,29 +104,33 @@ def train_end(self):
         pass
 
     def batch_begin(self):
-        self.batch_start = time.time()
+        if self.verbose == 2:
```

Comment:
Can we replace the hard coding with a constant?
It'll help in improving the readability as well as be less error prone if this needs further changes in the future.

Reply:
As per the discussion, I have added it as a constant so the end user can directly pass that as a parameter instead of number. Could you please have a look again? Thanks!